### PR TITLE
Docs to RTD

### LINKED ISSUE #66: Publish documentation on readthedocs.io
Currently this package has a custom way to create its documentation via a buildout recipe. I am not sure if it is still needed or even possible with readthedocs.io.

Currently the whole documentation is on the PyPI page. I think this could be given up in favour of readthedocs.io.

### PR DESCRIPTION
**Pull request by icemac**
When once complete fixes #66 

Documentation online see http://z3cform.readthedocs.io/en/docs-to-rtd/

### CODE REVIEW

**Comment by icemac** [comment]
The coverage decrease comes from a deleted line.

File: src/z3c/form/browser/checkbox.rst [hunk 1]
```diff
@@ -1,6 +1,5 @@
-==============
-CheckBoxWidget
-==============
+CheckBox Widget
```
Comment (mgedmin) [inline src/z3c/form/browser/checkbox.rst]: If the title refers to the class name it should stay "CheckBoxWidget".  If it refers to a checkbox widget, it should be "Checkbox Widget".  BiCapitalization looks wrong to me.

File: src/z3c/form/field.rst [hunk 2]
```diff
@@ -387,7 +387,7 @@ Next we need a form that specifies the fields to be added:
   ...     fields = field.Fields(IPerson)
   >>> personForm = PersonForm()
 
-For more details on how to define fields within a form, see ``form.txt``. We
+For more details on how to define fields within a form, see ``form.rst``. We
```
Comment (mgedmin) [inline src/z3c/form/field.rst]: TBH it would be better to use Sphinx's inter-document references for these.

File: tox.ini [hunk 3]
```diff
@@ -14,8 +14,16 @@ usedevelop = true
 basepython =
     python2.7
 commands =
-    coverage run -m zope.testrunner --test-path=src []
+    coverage run -m zope.testrunner --test-path=src --auto-color -vv []
```
Comment (mgedmin) [inline tox.ini]: Perhaps make it `coverage run -m zope.testrunner --test-path=src {posargs:--auto-color -vv}`, so people can override?

**Comment by icemac** [comment]
@mgedmin Thank you for your review.

**Pull request merged**